src/wallet/rpcwallet.cpp

```diff
@@ -308,22 +328,24 @@ UniValue setlabel(const JSONRPCRequest& request)
     }
 
     std::string label;
-    if (!request.params[1].isNull())
+    if (!request.params[1].isNull()) {
```

Comment:
It seems odd to me that `label` is marked as a required argument for `setlabel` but if it's not provided then we default to an empty string instead of throwing an error. If we want to keep the behavior then `label` should be marked as optional.

Reply:
I've updated the behaviour to match the help text: the rpc call will fail if no label is provided.